### PR TITLE
Adds new ColorUtils class to handle color alpha conversions.

### PR DESCRIPTION
**Pull request by chriscox**

### CODE REVIEW

File: src/lib/ColorUtils.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+export class ColorUtils {
+
+  /**
+   * Converts a color string or RgbaColor to an RGBA string.
+   * @param  {any}    color The color string or RgbaColor to convert.
+   * @return {string}       Returns the RGBA string.
+   */
+  static toRgbaString(color: any): string {
+    if (typeof color !== "string") {
+      // Convert alpha back from to int[0-255] to [0-1] float.
```
Comment (chuga) [inline src/lib/ColorUtils.ts]: remove first "to"

**Review by chuga — APPROVED** [review]
Any chance you might add a simple test to verify that the parsing and serialization is working as expected?

File: src/lib/ColorUtils.ts (hunk 1 above)
Comment (chriscox) [reply src/lib/ColorUtils.ts]: Done.

**Comment by chriscox** [comment]
Adds ColorUtils tests.

**Pull request merged**